src/release.js

```diff
@@ -507,6 +507,33 @@ module.exports = function taoExtensionReleaseFactory(params = {}) {
             }
         },
 
+        /**
+         * Verify that the version that we are going to release is valid
+         * - is the same on branch name and manifest
+         * - is bigger than current release branch version.
+         */
+        async verifyReleasingBranch() {
+            log.doing('Validating releasing branch.');
+
+            // Cross check releasing branch version with manifest version
+            await gitClient.checkout(data.releasingBranch);
+            const releasingBranchManifest = await taoInstance.parseManifest(`${data.extension.path}/manifest.php`);
+            if (compareVersions(releasingBranchManifest.version, data.version) == 0 ) {
```

Comment:
Please use `===` over `==` in such comparisons.

Reply:
yes. done.